Extensions/tweaks.js

```diff
@@ -27,10 +27,16 @@ XKit.extensions.tweaks = new Object({
 			value: true,
 			mobile_only: true
 		},
-		"wrap_tags": {
-			text: "Wrap tags for easier reading",
-			default: true,
-			value: true
+		"collapsible_tag_display": {
```

Comment:
Changing the preference name here leads to losing the preferences of people who already chose not to wrap tags.